data/data/bootstrap/baremetal/README.md

```diff
@@ -22,19 +22,33 @@ Relevant files:
 * **systemd/units/keepalived.service** - systemd unit file for `keepalived`.
   This runs `keepalived.sh` to generate the proper configuration from the
   template and then runs podman to launch `keepalived`.
+* **files/usr/local/bin/fletcher8** - 
+* **files/usr/local/bin/get_vip_subnet_cidr** - 
```

Comment:
Should these get descriptions? There are some already in the file, so wasn't sure:

fletcher8:

```
# Script that uses the fletcher8 algorithm to generate a hash from an input
# string. This is used to generate VRRP ids for use with Keepalived.
```

get_vip_subnet_cidr:

```
# Script to determine the network CIDR for a given VIP.
```

Reply:
Sure, I'll copy those descriptions